Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
 
-Currently, the IETF meets in North America, Europe, and Asia, approximately once a year in each region. The past few meetings have had about 1,200 attendees. There have been more than 80 IETF meetings so far, and a list of upcoming meetings is available on the IETF web pages, http://www.ietf.org/meeting/upcoming.html.
+Newcomers to IETF face-to-face meetings often expect them to be like other standards bodies, or like computer conferences. Fortunately, many new attendees get quite animated about how much fun they are having. On the other hand, IETFers can sometimes be surprisingly direct, sometimes verging on rude. To build a climate in which people of many different backgrounds are treated with dignity, decency, and respect IETF has both an anti-harsassment policy and an ombudsteam. (see [https://www.ietf.org/contact/ombudsteam/])
 
-Newcomers to IETF face-to-face meetings are often in a bit of shock. They expect them to be like other standards bodies, or like computer conferences. Fortunately, the shock wears off after a day or two, and many new attendees get quite animated about how much fun they are having. On the other hand, IETFers can sometimes be surprisingly rude, such as talking loudly during functions when someone is speaking at the microphone or pushing through a crowd to get to food or drinks. This type of anti-social behavior seems to be more common at IETF meetings than at computer conferences.
 3.1 Registration
 
-To attend an IETF meeting, you have to register and pay a registration fee. The meeting site and advance registration are announced about two months ahead of the meeting — earlier if possible. An announcement goes out via email to the IETF-announce mailing list, and information is posted on the IETF web site, http://www.ietf.org, that same day.
+To attend an IETF meeting, you have to register and pay a registration fee. The meeting site and advance registration are announced about two months ahead of the meeting — earlier if possible. An announcement goes out via email to the IETF-announce mailing list, and information is posted on the IETF web site, [https://www.ietf.org], that same day.
 
-You can register and pay on the web before the meeting, or in person at the meeting. To get a lower registration fee, you must pay by the early registration deadline (about one week before the meeting). The registration fee covers all of the week's meetings, the Sunday evening welcome reception (cash bar), daily continental breakfasts, and afternoon beverage and snack breaks.
+You can register and pay on the web before the meeting, or in person at the meeting. To get a lower registration fee, you must pay by the early registration deadline (about one week before the meeting). The registration fee covers all of the week's meetings, the Sunday evening welcome reception, and afternoon beverage and snack breaks.
```

Comment:
"about one week before the meeting" is changing for IETF 103

Reply:
Yes, this should be made consistent with the changes announced by the IAOC.

Reply:
s/one week/five weeks

Reply:
Five weeks is not correct. There are three registration windows: early bird, standard, and late. Early bird closes 7 weeks before the meeting. Standard closes 2 weeks before the meeting. Late is any time after standard.

Reply:
Changed: 

You can register and pay on the web before the meeting, or in person at the meeting. To get a lower registration fee, you must pay by the early-bird registration deadline (at least seven weeks before the meeting). The standard registration fee window closes 2 weeks before the meeting, if you register any time after that you will need to pay the fee for late registrants. The registration fee covers all of the week's meetings, the Sunday evening welcome reception, and afternoon beverage and snack breaks.